src/test/scala/chiselTests/AnalogIntegrationSpec.scala

```diff
@@ -31,15 +31,19 @@ class AnalogBlackBox(index: Int) extends BlackBox(Map("index" -> index)) {
   val io = IO(new AnalogBlackBoxIO(1))
 }
 
+trait AnalogBlackBoxModuleIntf extends Module {
```

Comment:
What's this for?

Reply:
It's an example that was hitting the `val io` deprecation. [`val mods` in `AnalogDUT`](https://github.com/chipsalliance/chisel3/blob/616256c35cb7de8fcd97df56af1986b747abe54d/src/test/scala/chiselTests/AnalogIntegrationSpec.scala#L66) relies on `io` being a field of `Module` (interestingly, it's actually inferring a structural type, `Module { val io: AnalogBlackBoxIO }`. The deprecation warning obviously became an error so I had to fix it and this is the canonical way.

Reply:
Ah, I see. Might be nice to add a quick comment with the trait about why it exists, though.

Reply:
:+1: on adding the scaladoc

Reply:
This is in `test` so not sure "ScalaDoc" is useful, but I will add a comment 🙂

Reply:
Done.